content/_policy/contact._fr.md

```diff
@@ -1,41 +1,41 @@
 
 help_center_content: >-
-  ## Trouver des réponses aux questions les plus fréquemment posées
+  ## Trouvez des réponses aux questions fréquemment posées
 
     {: .help-question-list}
   * [Je ne peux pas me connecter à mon compte](/fr/help/trouble-signing-in/overview/)
 
   * [J’ai besoin d’aide pour vérifier mon identité](/fr/help/verify-your-identity/overview/)
 
-  * [Je dois modifier mes informations ou gérer mon compte](/fr/help/manage-your-account/overview/)
+  * [J’ai besoin de modifier mes informations ou de gérer mon compte](/fr/help/manage-your-account/overview/)
 
-  * [Parcourir d’autres articles d’aide](/fr/help/)
-maintenance_window_content: Le centre de contact de Login.gov est actuellement en cours de maintenance <strong>%{start_time} au %{end_time}</strong>. Consultez les sujets courants ci-dessous pour obtenir de l'aide.
-phone_available_content: Vous pouvez appeler notre centre d'assistance à tout moment au (844) 875-6446.
-unplanned_outage_content: En raison d'une panne, nous ne pouvons pas traiter les demandes d'assistance en ligne.
+  * [Voir d’autres sujets d’assistance](/fr/help/)
+maintenance_window_content: Des travaux de maintenance sont actuellement en cours sur le centre de contact de Login.gov de <strong>%{start_time} à %{end_time}.</strong> Consultez ci-dessous certains sujets courants pour obtenir de l’aide.
+phone_available_content: Vous pouvez à tout moment appeler notre centre d’assistance au +1 (844) 875-644.
+unplanned_outage_content: En raison d’une panne, nous ne sommes pas en mesure de traiter les demandes d’assistance en ligne.
 partner_content: >-
-  ## Partenaire avec Login.gov
+  ## Devenez partenaire de Login.gov
 
-  Souhaitez-vous utiliser Login.gov dans votre agence? Veuillez [visiter le site web de nos partenaires](/partners/) ou [nous contacter](/partners/business-inquiries/).
+  Vous souhaitez utiliser Login.gov pour votre organisme ? Veuillez [visiter notre site Web pour les partenaires](/partners/) ou [nous contacter](/partners/business-inquiries/).
 
 report_issue_content: >-
   ## Signaler un problème
 
-  Si vous souhaitez signaler un problème, veuillez consulter [notre politique en matière de divulgation des vulnérabilités](https://handbook.tts.gsa.gov/general-information-and-resources/tech-policies/responding-to-public-disclosure-vulnerabilities/) et nous contacter en utilisant [notre formulaire de divulgation des vulnérabilités](https://docs.google.com/forms/d/e/1FAIpQLScuo4xCzBlpLnoq7-bDAVAxtJci03by7S-Q-Z_JUBDloK01QA/viewform).
+  Pour nous signaler un problème, veuillez consulter notre [politique de divulgation des vulnérabilités](https://handbook.tts.gsa.gov/general-information-and-resources/tech-policies/responding-to-public-disclosure-vulnerabilities/ "Follow link") et nous contacter à l'aide de notre [formulaire](https://docs.google.com/forms/d/e/1FAIpQLScuo4xCzBlpLnoq7-bDAVAxtJci03by7S-Q-Z_JUBDloK01QA/viewform "Follow link") prévu à cet effet.
```

Comment:
Is "Follow link" a title here? Should it be removed or translated?

Reply:
Hm, it's in the English version, so it was left in here. I removed it from this set of changes (though I left the English one).